src/overlays/actors/ovl_En_Dodongo/z_en_dodongo.c

```diff
@@ -136,97 +280,839 @@ static DamageTable D_808792D4 = {
     ICHAIN_F32(targetArrowOffset, 1400, ICHAIN_STOP),
 };
 
-#endif
+void EnDodongo_Init(Actor* thisx, GlobalContext* globalCtx) {
+    static s32 D_80879308[] = {
+        0x00000002, 0x00080000, 0xFFFFFFFF, 0xFFFFFF40, 0xFFFFFF00, 0xFFFFFF00, 0x08000000, 0x00000000, 0x00000000,
+    };
+
+    EnDodongo* this = THIS;
+    s32 i;
+
+    Actor_ProcessInitChain(&this->actor, sInitChain);
+    this->unk_32C.r = 255;
+    this->unk_32C.g = 255;
+    this->unk_32C.a = 200;
+    this->unk_330.r = 255;
+    this->unk_330.g = 10;
+    this->unk_330.a = 200;
+    Math_Vec3f_Copy(&this->unk_314, &D_801C5DB0);
+    ActorShape_Init(&this->actor.shape, 0.0f, func_800B3FC0, 48.0f);
+    SkelAnime_Init(globalCtx, &this->skelAnime, &D_06008318, &D_06004C20, this->jointTable, this->morphTable, 31);
+    CollisionCheck_SetInfo(&this->actor.colChkInfo, &sDamageTable, &sColChkInfoInit);
+    Collider_InitAndSetJntSph(globalCtx, &this->collider2, &this->actor, &sJntSphInit2, this->collider2Elements);
+    Collider_InitAndSetJntSph(globalCtx, &this->collider1, &this->actor, &sJntSphInit1, this->collider1Elements);
+    Collider_InitAndSetJntSph(globalCtx, &this->collider3, &this->actor, &sJntSphInit3, this->collider3Elements);
+
+    for (i = 0; i < ARRAY_COUNT(this->collider2Elements); i++) {
+        this->collider2.elements[i].info.elemType = ELEMTYPE_UNK2;
+        this->collider2.elements[i].info.bumper.dmgFlags = 0x77C34FE6;
+    }
+
+    Effect_Add(globalCtx, &this->unk_338, 2, 0, 0, &D_80879308);
+    if (!this->actor.params) {
+        Actor_SetScale(&this->actor, 0.01875f);
+        this->unk_334 = 1.0f;
+    } else {
+        Actor_SetScale(&this->actor, 0.0375f);
+        this->actor.colChkInfo.health = 12;
+        this->unk_334 = 2.0f;
+
+        for (i = 0; i < ARRAY_COUNT(this->collider1Elements); i++) {
+            this->collider1.elements[i].dim.modelSphere.radius *= 2;
+            this->collider1.elements[i].info.toucher.damage *= 2;
+        }
+
+        for (i = 0; i < ARRAY_COUNT(this->collider2Elements); i++) {
+            this->collider2.elements[i].dim.modelSphere.radius *= 2;
+        }
+
+        for (i = 0; i < ARRAY_COUNT(this->collider3Elements); i++) {
+            this->collider3.elements[i].dim.modelSphere.radius *= 2;
+        }
+    }
+
+    func_808773C4(this);
+}
+
+void EnDodongo_Destroy(Actor* thisx, GlobalContext* globalCtx) {
+    EnDodongo* this = THIS;
+
+    Effect_Destroy(globalCtx, this->unk_338);
+    Collider_DestroyJntSph(globalCtx, &this->collider2);
+    Collider_DestroyJntSph(globalCtx, &this->collider3);
+    Collider_DestroyJntSph(globalCtx, &this->collider1);
+}
+
+void func_80876930(EnDodongo* this, GlobalContext* globalCtx, Vec3f* arg2) {
+    static Color_RGBA8 D_8087932C = { 0xFA, 0xFA, 0xFA, 0xFF };
+    static Color_RGBA8 D_80879330 = { 0xB4, 0xB4, 0xB4, 0xFF };
+    static Color_RGBA8 D_80879334 = { 0xAA, 0x82, 0x5A, 0xFF };
+    static Color_RGBA8 D_80879338 = { 0x64, 0x3C, 0x14, 0xFF };
```

Comment:
```suggestion
    static Color_RGBA8 D_8087932C = { 250, 250, 250, 255 };
    static Color_RGBA8 D_80879330 = { 180, 180, 180, 255 };
    static Color_RGBA8 D_80879334 = { 170, 130, 90, 255 };
    static Color_RGBA8 D_80879338 = { 100, 60, 20, 255 };
```
Colours are one thing we always do in dec (others are things like timers, health, and collider dimensions).